src/app.ts

```diff
@@ -179,13 +187,25 @@ export class App {
     };
 
     this.world.scene = sceneEl.object3D;
+    const scene = sceneEl.object3D;
+
+    // We manually call scene.updateMatrixWolrd in mainTick
+    scene.autoUpdate = false;
+
+    if (enablePostEffects) {
+      this.fx = createEffectsComposer(canvas, renderer, camera, scene, sceneEl, this.store);
+    } else {
+      (sceneEl as any).addEventListener("rendererresize", function ({ detail }: { detail: DOMRectReadOnly }) {
+        renderer.setSize(detail.width, detail.height, true);
+      });
+    }
```

Comment:
Does here mean render resize handling is done in the effect composer if post effects is enabled? (Maybe adding comments is good)